src/pages/components/Sidebar/SidebarContent.tsx

```diff
@@ -42,11 +44,20 @@ const SidebarContent: React.FC<SidebarProps> = function SidebarContent({ onClose
       </SidebarItem>
 
       {LinkItems.map((link) => (
-        <SidebarItem key={link.name} href={link.href} icon={link.icon} _hover={{ bg: 'blue.400', color: 'white' }}>
-          <Text fontSize="md" fontFamily="monospace" fontWeight="medium" paddingLeft={3}>
-            {link.name}
-          </Text>
-        </SidebarItem>
+        <Link key={link.name} href={link.href} passHref>
+          <SidebarItem
+            href={link.href}
+            icon={link.icon}
+            _hover={{ bg: 'blue.400', color: 'white' }}
+            color={router.pathname === link.href ? 'white' : 'gray.800'}
+            activeBgColor={router.pathname === link.href ? 'blue.600' : 'transparent'}
+            activeTextColor={router.pathname === link.href ? 'white' : 'gray.800'}
```

Comment:
nit: create a top-level variable called `isCurrentHref` so you can replace all the `router.pathname === link.href`